js/apps/events-page/services/dateUtilityService.js

```diff
@@ -3,11 +3,11 @@
 
 	const dateUtilityService = () => {
 		const addDays = (dateOrString, daysToAdd) => {
-			const date = typeof dateOrString === 'string' ? new Date(dateOrString) : dateOrString;
+			const date = typeof dateOrString === 'string' ?
+				new Date(dateOrString) :
+				dateOrString;
```

Comment:
Personally I prefer ternary notation at the beginning of the line but that's splitting hairs.

Reply:
@ascire There was a reason i did it this way, and now I don't know why. I originally thought this was an error from the IDE, but I justed tested it and no dice. Mike prefers taht way too, so I am going to update our guidelines.